src/features/bookings/components/BookingDetailsTicketContent.native.test.tsx

```diff
@@ -26,4 +26,25 @@ describe('BookingDetailsTicketContent', () => {
       expect(queryByText(booking.token)).toBeFalsy()
     })
   })
+
+  describe('proDisableEventsQrcode & QR code display', () => {
+    it('should display the QR code when proDisableEventsQrcode is false', () => {
+      const booking = bookingsSnap.ongoing_bookings[0]
+      booking.stock.offer.isDigital = false
+      const { getByTestId } = render(
+        <BookingDetailsTicketContent booking={booking} proDisableEventsQrcode={false} />
+      )
+      getByTestId('qr-code')
+    })
+
+    it('should not display the QR code when proDisableEventsQrcode is true', () => {
+      const booking = bookingsSnap.ongoing_bookings[0]
+      booking.stock.offer.isDigital = false
+      const { getByTestId } = render(
+        <BookingDetailsTicketContent booking={booking} proDisableEventsQrcode={true} />
+      )
+
+      expect(() => getByTestId('qr-code')).toThrow()
```

Comment:
il me semble qu'on peut faire plus simplement

```suggestion
      expect(queryByTestId('qr-code')).toBeFalsy()
```